components/core/config/config.ts

```diff
@@ -47,6 +47,7 @@ export interface NzConfig {
   tree?: TreeConfig;
   treeSelect?: TreeSelectConfig;
   typography?: TypographyConfig;
+  global?: GlobalConfig;
```

Comment:
As far as I know, this config is only applicable to specific components such as **Notification** and **Message**. I recommend to change this to become one of their services' API parameters.

Reply:
@HDaghash